0.7-final

balloob opened the pull request:
Changelog since RC-1 (#290):
- Bugfixes
- documentation updates
- deprecate `scripts/update`
- updated console output to help debug problems.
- Device tracker: new platform aruba (@michaelarnauts)
- Switch: Wemo maker support (@pavoni)

Commit:
Add standby state to WeMo Insight Switch, and add WeMo Maker

Commit:
Fix blank line regression

Commit:
Add defult methods to base class for switch_state and sensor_state

balloob committed:
Fix download link in setup.py

balloob committed:
Update README

balloob committed:
Add --version command line argument

balloob committed:
Upgrade media_player.cast dependency

rmkraus committed:
Added test for conversation and introduction components.

rmkraus committed:
Added another conversation test.

rmkraus committed:
Fixed remote tests.

rmkraus committed:
Restructured conversation tests.

renekliment committed:
support for MPD password authorization

renekliment committed:
fixes MPD play/pause

balloob committed:
Prevent init screen from shifting

balloob committed:
Update demo component

balloob committed:
Merge pull request #294 from renekliment/mpd-password-authorization

support for MPD password authorization

Commit:
Merge remote-tracking branch 'balloob/dev' into feature/enhance_wemo

Commit:
Update pywemo to 0.3

Commit:
Update pywemo to 0,3

balloob committed:
Merge pull request #295 from renekliment/mpd-pause-fix

fixes MPD play/pause

balloob committed:
Fix components being loaded twice

balloob committed:
Bugfix release frontend

rmkraus committed:
Fixed importing of dependency for map tracker.

rmkraus committed:
Fixed device tracker to install scanner requirements.

rmkraus committed:
Fixed device tracker to install scanner requirements.

balloob committed:
Update screenshot

balloob committed:
Upgrade frontend

balloob committed:
Allow spaces in group setup string

balloob committed:
Improved console output for new users

balloob committed:
Deprecate scripts/update

rmkraus committed:
Removed bogus intro test.

balloob committed:
Merge pull request #291 from rmkraus/more_tests

Adding additional tests

balloob committed:
Migrate nmap_tracker to use different nmap lib

balloob committed:
style + dependency fix

balloob committed:
Fix broken conversation test

michaelarnauts committed:
Add Aruba Access Point device tracker

Commit:
Refactor wemo standby state slightly

Commit:
Fix typos

Commit:
Fix bug, use constants for states.

Commit:
Split comment to keep line short

balloob committed:
Update efergy descriptin in README

balloob committed:
Merge pull request #296 from pavoni/feature/enhance_wemo

Feature/enhance wemo

Commit:
Remove debug statements!

balloob committed:
Merge pull request #302 from michaelarnauts/dev

Add Aruba Access Point device tracker

balloob committed:
Update frontend with latest changes

balloob committed:
Add aruba device tracker to coveragerc

balloob committed:
Add Python 3.4 check back

rmkraus commented on homeassistant/components/media_player/cast.py:
```diff
@@ -19,7 +19,7 @@
     MEDIA_TYPE_MUSIC, MEDIA_TYPE_TVSHOW, MEDIA_TYPE_VIDEO)
 
-REQUIREMENTS = ['pychromecast==0.6.10']
+REQUIREMENTS = ['pychromecast==0.6.12']
```
:+1:

balloob committed:
Make nmap work in Docker

rmkraus commented:
Looks good to me. I'm going to fix the daemon scripts right now. It might be good to include those in rc2.

rmkraus committed:
Merging balloob/dev into dev.

balloob committed:
Merge pull request #300 from balloob/nmap-fix

Migrate nmap_tracker to use different nmap lib

balloob committed:
Style fix for device tracker

rmkraus committed:
Added core daemon function as flags.

Created three additional flags for the hass command:
-v -  Toggle verbose log file output
—pid-file -  Specify PID file path
—daemon -  Launch as daemon (nix only)

The core now binds to SIGQUIT on nix systems to trigger a clean
shutdown.

Modified HTTP server to write logging messages through the logging
module.

rmkraus committed:
Merge balloob/dev with rmkraus/daemon

rmkraus committed:
Daemon handling cleanup.

More cleanly handling whether a PID file should be checked.
Fatal Error on PID file IOError.
Pylint fix to bootstrap.

balloob committed:
Speed up tests

rmkraus committed:
Updated Debian daemon script.

rmkraus committed:
Cleanup PID checking logic and write PID logic.

rmkraus committed:
Fixes to new Debian daemon script.

balloob committed:
Merge pull request #303 from rmkraus/daemon

Updating Daemon Code
